vivisect/analysis/i386/calling.py

```diff
@@ -111,6 +111,8 @@ def analyzeFunction(vw, fva):
         api = buildFunctionApi(vw, fva, emu, emumon)
 
     rettype,retname,callconv,callname,callargs = api
+    if callconv == 'unkcall':
```

Comment:
Do we really want to bail on all unkcall? The emulation pass still ends up filling in a lot of knowledge even if we dont know the exact state at entry/exit right?